packages/playwright-ct-angular/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@jscutlery/playwright-ct-angular",
   "version": "0.3.0",
   "dependencies": {
+    "@babel/core": "~7.23.0",
```

Comment:
Shouldn't we update playwright peer dependencies as well?

Reply:
Oh! Good point! Actually, they should be dependencies. I'll fix this in another PR.